guides/scripts/generate-hammer-reference.sh

```diff
@@ -153,9 +151,13 @@ cat "$SRC" | while read line ; do
             [ -n "$DEBUG" ] && echo -e "I: [cmd-3]: $line"
             id_core=`echo -e $line | sed 's/^### //' | sed 's/ /-/g'`
             id=[id=\"$id_core\"]
-            heading=`echo -e $line | sed 's/^### hammer/==/'`
+            filename=ref_$id_core.adoc
```

Comment:
What does "core" mean? Does that mean that we have modules with two underscores?

I would prefer a second/different prefix if possible. As a maintainer, I would probably want an easy way to exclude those automatically generated files when I make tree-wide changes.

Reply:
Oh. Didn't realize that. Then it's OK as is. Alternative suggestion to emphasise that it's the name of a variable:

```suggestion
            filename=ref_${id_core}.adoc
```